.gitignore

```diff
@@ -14,6 +14,7 @@ node_modules
 bundle.js
 tailwind.css
 *.pem
+*/package-lock.json 
```

Comment:
suggestion (blocking): please revert. the lock files are important for keeping our dependencies consistent.

Reply:
done